### ISSUE TITLE
Consider dropping requirement for Tool Data Sink when only one host involved

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#2191]
If only one host is registered, and that host is the same as the host on which `pbench-tool-meister-start` is executed (the "controller") then the Tool Data Sink process is not needed when there are only transient tools registered.

**Comment by webbnh** [comment]
This is true, but why is it important to make this distinction?  What do we or the user gain by omitting the TDS?  Is it worth the complication of providing a separate "mode" for this special case?
